### PR TITLE
✨ Implement NFT book storefront

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

**Review by AuroraHuang22 — APPROVED** [review]

File: src/components/NFTCover.vue [hunk 1]
```diff
@@ -83,24 +98,33 @@ export default {
...
+            'h-full min-h-[290px] max-w-[200px]': this.isNftBook,
           },
         ],
         loading: 'lazy',
         ...this.$attrs,
       };
     },
     resizedSrc() {
-      return getLikeCoResizedImageUrl(this.src, this.size);
+      return getImageResizeAPI(this.src, { width: this.size });
```
Comment (williamchong) [inline src/components/NFTCover.vue:112]: `getLikeCoResizedImageUrl` only works for image served by like.co, while `getImageResizeAPI` was for old likerland og
Maybe we should merge both into one function

File: src/components/Footer.vue [hunk 2]
```diff
@@ -38,7 +38,7 @@
           <a class="hover:underline" target="_blank" href="https://like.co">{{ $t('footer_nav_about_likecoin') }}</a>
         </div>
         <div class="grid grid-flow-row gap-y-[16px]">
-          <NuxtLink class="hover:underline" :to="localeLocation({ name: 'writing-nft' })">{{ $t('footer_nav_writing_nft') }}</NuxtLink>
+          <NuxtLink class="hover:underline" :to="localeLocation({ name: 'index' })">{{ $t('footer_nav_writing_nft') }}</NuxtLink>
```
Comment (williamchong) [inline src/components/Footer.vue:41]: Should this go to `writing-nft-about` instead?

File: src/pages/index.vue [hunk 3]
```diff
@@ -1,7 +1,314 @@
...
+      'pb-[132px]',
+    ]"
+  >
+    <section
+      :class="[
+        'w-full',
+        'max-w-[960px]',
+      ]"
+    >
+      <h2 class="text-[#3AB7A2] text-[48px] font-proxima font-[600]">{{ $t('home_section_book_title') }}</h2>
```
Comment (williamchong) [inline src/pages/index.vue:16]: Have book and article in left and right tabs seems better, the current page is too long and articles(WNFT) is too far down below

Comment (nwingt) [reply src/pages/index.vue:16]: Updated as @joshkiu design

image
image
image

File: src/components/NFTCover.vue (hunk 1 above)
Comment (williamchong) [reply src/components/NFTCover.vue:116]: For writing nfts they might, since uri override exists

Comment (nwingt) [reply src/components/NFTCover.vue:116]: So the criteria for `getLikeCoResizedImageUrl` is URL includes `like.co`?

Comment (williamchong) [reply src/components/NFTCover.vue:116]: Yes we can do that for now

**Review by williamchong — APPROVED** [review]
Please fix lint warning before merge

**Pull request merged**